packages/web-frontend/src/screens/desktop/RootScreen/MainPage.js

```diff
@@ -51,28 +51,34 @@ const MapContainer = styled.div`
 
 const MainPage = ({ enlargedDialogIsVisible, isLoading, sidePanelWidth }) => {
   const { isCustomLandingPage } = useCustomLandingPages();
-  if (isCustomLandingPage) {
```

Comment:
If it's ok I would prefer to have the landing page as it's own template with it's own EnvBanner and TopBar as it will save a bunch of conditional statements on isCustomLandingPage. I always find conditionals are hard to read and don't scale well.